SportHub.API/Controllers/AuthController.cs

```diff
@@ -0,0 +1,97 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SportHub.Business;
+using SportHub.Data;
+using SportHub.Business;
+
+namespace SportHub.Controllers;
+
+[AllowAnonymous]
+[ApiController]
+[Route("[controller]")]
+public class AuthController : ControllerBase
+{
+    private readonly IUserService _userService;
+    private readonly IJwtService _jwtService;
+    
+    public AuthController(IUserService userService, IJwtService jwtService)
+    {
+        _userService = userService;
+        _jwtService = jwtService;
+    }
+
+    [HttpPost("login")]
+    [AllowAnonymous]
+    public IActionResult LogIn(string email, string password)
+    {
+        var user = _userService.GetUserByEmailAsync(email).Result;
```

Comment:
You should never use Result on async method

Reply:
I would sugest GetUser should return user only if email and pasword match, otherwise null